package.json

```diff
@@ -154,42 +154,42 @@
     "morgan": "~1.9.1",
     "multer": "~1.4.1",
     "ng-file-upload": "~12.2.13",
-    "nodemailer": "~4.7.0",
+    "nodemailer": "~5.1.1",
```

Comment:
DNS resolver has some changes, should be good. Will need to double check in production.